cypress/start.sh

```diff
@@ -19,7 +19,7 @@ for i in "$@" ; do
     fi
 done
 
-if [[ $GUI == "true" ]] || [[ $CI != "true" ]] ; then
+if [[ $GUI == "true" ]] ; then
```

Comment:
cypress should be enabled; this just makes it possible to run the tests locally without the GUI

Reply:
Are you sure you want to take that out?  Original logic was to enable CI to run tests tagged signup followed by others, whereas locally, we didn't run the signup test from command line (we could from the GUI).  Maybe things have moved on since, but worth another check?

Reply:
I think the reason I removed this was because, if I set CI to "true" locally, the tests change how they run, and start using CI-specific endpoints. This meant I couldn't run the tests from the command-line locally using start.sh direct.

I think I will put it back in, since most of the time I've been using the GUI now the tests I'm working on don't need S3Monitor.

Reply:
Would it make sense to change this to:

```
if [[ $CI == "true" ]] ; then
    # Its headless (used in CircleCI) so run the signup test first followed by all others
    echo "Running Cypress headless"
    ./node_modules/.bin/cypress-tags run -e TAGS='@SignUp'
    ./node_modules/.bin/cypress-tags run -e TAGS='not @SignUp'
else
    echo "Running Cypress"
    # pass supplied args to cypress
    CYPRESS_CMD="cypress $@"
    echo $CYPRESS_CMD
    $CYPRESS_CMD
fi
```

?

(I get confused by the logic in the original if condition.)

Reply:
I had another look at this and did switch the conditional, but added a CYPRESS_headless env var. Setting CI=true locally causes S3Monitor to use the wrong credentials (it attempts to use the CI ones).